js/customize-concurrency.js

```diff
@@ -25,8 +25,18 @@
 	 */
 	component.init = function() {
 		api.bind( 'ready', function() {
-			api.bind( 'change', function() {
-				// todo: update concurrency data
+			component.data['settingModifiedTimes'] = {};
+
+			api.each( function( setting ) {
+				component.data.settingModifiedTimes[ setting.id ] = new Date().valueOf();
+			} );
+
+			api.bind( 'add', function( setting ) {
+				component.data['settingModifiedTimes'][ setting.id ] = new Date().valueOf();
+			} );
+
+			api.bind( 'change', function( setting ) {
+				component.data['settingModifiedTimes'][ setting.id ] = new Date().valueOf();
```

Comment:
I guess that wouldn't work when I add snapshots, huh? I just need to refactor so I am not burying so deep.